app/api/novu/route.ts

```diff
@@ -1,12 +1,4 @@
 import { serve } from "@novu/framework/next";
-import { welcomeOnboardingEmail } from "../../echo/client";
-import { Client } from '@novu/framework';
+import { welcomeOnboardingEmail } from "../../novu/workflows";
 
-const client = new Client({
-    /**
-     * Enable this flag only during local development
-     */
-    strictAuthentication: false,
-});
-
-export const { GET, POST, PUT, OPTIONS } = serve({ client, workflows: [welcomeOnboardingEmail] });
+export const { GET, POST, PUT, OPTIONS } = serve({ workflows: [welcomeOnboardingEmail] });
```

Comment:
`PUT` is not used today, let's remove to avoid confusion.

```suggestion
export const { GET, POST, OPTIONS } = serve({ workflows: [welcomeOnboardingEmail] });
```